features/cli/deploy.feature

```diff
@@ -676,9 +676,14 @@ Feature: deployment related features
     Given I wait until number of replicas match "0" for replicationController "hooks-1"
     Then I wait for the "hooks-2" rc to appear
     And I wait until number of replicas match "1" for replicationController "hooks-2"
-    When I get project pod
-    Then the output should match:
-      | hooks-2.*Running |
+    Given I wait up to 60 seconds for the steps to pass:
+      """
```

Comment:
Can you please remove the indent to align the `"""` with the above line. There is no need to indent here.

Reply:
Updated